src/components/snapshot/AuthDomain.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Box, FormLabel, Link, styled } from '@mui/material';
+import { ManagedGroupMembershipEntry } from 'src/models/group';
+import { AppDispatch } from 'src/store';
+import { TdrState } from 'src/reducers';
+import { useOnMount } from 'src/libs/utils';
+import { LaunchOutlined } from '@mui/icons-material';
+import { connect } from 'react-redux';
+import { getUserGroups } from 'src/actions';
+import JadeDropdown from '../dataset/data/JadeDropdown';
+
+const JadeLink = styled('span')(({ theme }) => theme.mixins.jadeLink);
+
+type AuthDomainProps = {
+  dispatch: AppDispatch;
+  userGroups: Array<ManagedGroupMembershipEntry>;
+  setParentAuthDomain: (domain: string) => void;
+};
+
+function AuthDomain({ dispatch, userGroups, setParentAuthDomain }: AuthDomainProps) {
+  const [selectedAuthDomain, setSelectedAuthDomain] = React.useState<string | undefined>(undefined);
+
+  useOnMount(() => {
+    dispatch(getUserGroups());
+  });
+
+  return (
+    <>
+      <FormLabel sx={{ fontWeight: 700, color: '#333f52' }} htmlFor="authorization-domain">
+        Authorization Domain
+        <span style={{ fontWeight: 400, fontStyle: 'italic' }}> - (optional)</span>
+      </FormLabel>
+      <Box sx={{ mb: 1 }}>
+        Authorization Domains restrict data access to only specified individuals in a group and are
+        intended to fulfill requirements you may have for data governed by a compliance standard,
+        such as federal controlled-access data or HIPAA protected data. They follow all snapshot
+        copies and cannot be removed. For more details, see{' '}
+        <Link
```

Comment:
This uses `Link` but the other use of `JadeLink` uses `<a href`. Is there a reason to have different forms in these two cases?

Could a more general component be defined that parameterizes the `href` and prompt text and shares the rest of the code in both places?

Reply:
I don't have a strong opinion on this, perhaps something we can discuss as a team?

Reply:
I created a ticket for this since it's inconsistent across the repo.